rpc/clients/typescript/src/types.ts

```diff
@@ -182,7 +182,7 @@ export interface LatestBlock {
 
 export interface GetStatsResponse {
     Version: string;
-    PubSubTopic: string;
+    PubSubTopics: string[];
```

Comment:
Maybe we should also separate the `GetStatsResponse` into `PublishTopics` and `SubscribeTopics` in order to provide a clearer picture of what is really going on under the hood.